docs/shared/concepts/how-caching-works.md

```diff
@@ -54,11 +54,11 @@ For more information on the different types of inputs and how to configure input
 
 ## What is Cached
 
-Nx works on the process level. Regardless of the tools used to build/test/lint/etc.. your project, the results are cached.
+Nx cache works on the process level. Regardless of the tools used to build/test/lint/etc.. your project, the results are cached. This includes:
 
-It collects terminal output when running tasks. All the terminal output is cached and then replayed during a cache hit.
-
-Nx can also cache the files generated by a task. The list of files/folders is listed in the `outputs` property of the project's `package.json` or `project.json`:
+- **Terminal output:** The terminal output generated when running a task. This includes logs, warnings, and errors.
+- **Task artifacts:** The output of a task defined in the [`outputs` property of your project configuration](/recipes/running-tasks/configure-outputs). For example the build output, test results, or linting reports.
```

Comment:
```suggestion
- **Task artifacts:** The output files of a task defined in the [`outputs` property of your project configuration](/recipes/running-tasks/configure-outputs). For example the build output, test results, or linting reports.
```

Reply:
Clarifying that artifacts means files.  Otherwise you could read this as the build outputs, test results and linting reports are also terminal outputs.